R/OptimizerChain.R

```diff
@@ -0,0 +1,159 @@
+#' @title Run Optimizers Sequentially
+#'
+#' @include Optimizer.R
+#' @name mlr_optimizers_chain
+#'
+#' @description
+#' `OptimizerChain` allows to run different [Optimizer]s sequentially.
+#'
+#' For each [Optimizer] an (optional) additional [Terminator] can be specified
+#' during construction.
```

Comment:
Maybe you can explain here how this additional terminator is used. The explanation that is nested in the below paragraph is hard to understand.